src/helix-ui/styles/core/hx-busy.less

```diff
@@ -0,0 +1,19 @@
+hx-busy {
+  animation-play-state: running;
+  animation: hx-spin 0.8s linear infinite;
+  border-color: transparent currentColor currentColor;
+  border-radius: 1em;
+  border-style: solid;
+  border-width: 2px;
+  box-sizing: border-box;
+  display: inline-block;
+  height: 1em;
+  transform-box: border-box;
```

Comment:
Zoom review: Add comments on 11 and 12 re: why this was added (IE11 and a bit of Edge reasons)